benchmarks_test.go

```diff
@@ -0,0 +1,85 @@
+package blake2b
```

Comment:
yes, of course, done.

> On 05 Jul 2016, at 12:18, Harshavardhana <notifications@github.com> wrote:
> 
> In benchmarks_test.go <https://github.com/minio/blake2b-simd/pull/12#discussion_r69539260>:
> 
> > @@ -0,0 +1,85 @@
> > +package blake2b
> Can you add a license header?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/minio/blake2b-simd/pull/12/files/b16d81524c5d4a0b8dba55c935b3d958f66f7125#r69539260>, or mute the thread <https://github.com/notifications/unsubscribe/ABgMTUd3zDRz6RM1DAnYVXXDipMgKBG5ks5qSi97gaJpZM4JE9ct>.
>